ocaml/xapi/xapi_pusb.ml

```diff
@@ -29,6 +29,62 @@ let create ~__context ~uSB_group ~host ~other_config ~path
   debug "PUSB ref='%s' created" (Ref.string_of pusb);
   pusb
 
+let scan_start ~__context usbs =
+  let host = Helpers.get_localhost ~__context in
+  let known_usb =
+    Db.PUSB.get_all_records ~__context
+    |> List.filter (fun (rf, rc) -> rc.API.pUSB_host = host)
+    |> get_known_usb
+  in
+  let local_usb = get_local_usb usbs in
+  (* Create the newly added pusbs *)
+  USBSet.iter (fun s -> let self = create ~__context ~uSB_group:(Ref.null) ~host ~other_config:[] ~path:s.USB.path ~vendor_id:s.USB.vendor_id
```

Comment:
This is a matter of taste: when lines are very long (like here on GitHub), I believe code is more readable and easier to changes when each labeled parameter is on a line by its own:
```
 USBSet.iter (fun s -> let self = create ~__context 
  ~uSB_group:(Ref.null) 
  ~host 
  ~other_config:[] 
  ~path:s.USB.path 
  ~vendor_id:s.USB.vendor_id
  ~vendor_desc:s.USB.vendor_desc 
  ~product_id:s.USB.product_id 
  ~product_desc:s.USB.product_desc 
  ~serial:s.USB.serial
  ~version:s.USB.version 
  ~description:s.USB.description 
  in
  let group = Xapi_pusb_helpers.find_or_create ~__context self in
  Db.PUSB.set_USB_group ~__context ~self ~value:group
```

Reply:
Ok, Thanks @lindig